tools_and_services/stack_guideline.md

```diff
@@ -2,46 +2,82 @@
 Also, to make it easier to on-board new employees.
 
+## Programming languages
 
-## Frontend
-- [React](https://reactjs.org/)
-- [React Native](https://reactnative.dev/)
+- JavaScript (TypeScript)
+- Ruby
+- C#
 
-#### Testing
-- [Jest](https://jestjs.io/)
-- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro)
+## Managing DBs and runtimes
+- [asdf](https://asdf-vm.com/) to easily install the required runtimes for most of our projects
+- [docker-compose](https://docs.docker.com/compose/) to easily get the DBs started - using the right versions
 
-#### CI
-- [GitHub Actions](https://github.com/features/actions)
+Dotnet has it's own built-in sdk manager.
 
-#### CD
-- [Heroku](https://heroku.com/)
+## Frameworks
 
-#### Monitoring
-- [Rollbar](https://elements.heroku.com/addons/rollbar): Error-tracking as an add-on for Heroku
-- [Librato](https://elements.heroku.com/addons/librato): Platform monitoring as an add-on for Heroku
+### Web apps
 
-## Backend
-- [Ruby on Rails](https://rubyonrails.org/) with the [Muffi template](https://github.com/abtion/muffi)
-- [.NET](https://dotnet.microsoft.com/en-us/) with the [MuffiNet template](https://github.com/abtion/muffi.net)
+- [Ruby on Rails](https://rubyonrails.org/) using our [rails template](https://github.com/abtion/muffi)
+- [.NET](https://dotnet.microsoft.com/en-us/) using our [.NET template](https://github.com/abtion/muffi.net)
 
-#### Database
-- [PostgreSQL](https://www.postgresql.org/)
+### Websites
+- [Strapi](https://strapi.io/) as CMS
+- [Next.js](nextjs.org/) as frontend (with or without CMS)
+- [Wordpress](https://wordpress.com/)*
 
-#### CI
-- [GitHub Actions](https://github.com/features/actions)
+\* Wordpress has it's own section further down the page as it's a whole separate ecosystem
+
+### Mobile apps
+- [React Native](https://reactnative.dev/)
 
-#### CD
-- [Heroku](https://heroku.com/)
+## Tooling
 
-#### Preferred Heroku Add-ons
-— [Rollbar](https://elements.heroku.com/addons/rollbar)
-— [Heroku Postgres](https://elements.heroku.com/addons/heroku-postgresql)
+### Hosting
+- PaaS (Platform as a Service)
+  - [heroku](https://heroku.com)
+  - [scalingo](https://scalingo.com/)
+- Storage
+  - [Amazon S3](https://aws.amazon.com/s3/)
+  - [Scaleway Object Storage](https://www.scaleway.com/en/object-storage/)
+
+### CI
+- [GitHub Actions](https://github.com/features/actions)
+
+### CD
+- Through the chosen PaaS (heroku / scalingo)
+- Or github actions
+
+### Monitoring
+- [Rollbar](https://elements.heroku.com/addons/rollbar)
+- [Librato](https://elements.heroku.com/addons/librato)
+- [Papertrail](https://www.papertrail.com/)
+
+### Emails
+- [Sendinblue](https://www.sendinblue.com/)
+
+### Testing
+- [Jest](https://jestjs.io/) for JavaScript/TypeScript
+- [PlayWright](https://playwright.dev/) for JavaScript/TypeScript E2E
```

Comment:
```suggestion
- [Playwright](https://playwright.dev/) for JavaScript/TypeScript E2E
```

Reply:
Isn't Playwright for more than just JavaScript/TypeScript?

Reply:
It is, but at least the C# implementation is very poor compared to JavaScript. I don't think we can expect the other languages to be a good experience.